### PR TITLE
Environment & Content Build Process Docs

### PR DESCRIPTION
**Pull request by jonpugh**
## Description

Improving documentation about environments and adding docs about the WEB Build process.

### CODE REVIEW

**Comment by jonpugh** [comment]
Not quite done but ready for some review.

Spots that need work are marked with TODO.

File: READMES/environments.md [hunk 1]
```diff
@@ -1,19 +1,63 @@
-# Environments
+# Environments & the Content Build Process
 
-Domains for this application are below, they both correspond to a frontend (FE) domain that consumes data from the CMS via GraphQL API endpoint at /graphql:
+In order to ensure quality and prevent problems from going to production, multiple
```
Comment (jefflbrauer) [inline READMES/environments.md]: To enable end to end quality analysis, from the VA.gov CMS through to the public facing website, new feature review, and stakeholder demonstrations, multiple joined CMS/WEB environments may be created by any user with access to devshop.

File: READMES/environments.md [hunk 2]
```diff
@@ -1,19 +1,63 @@
...
+The *WEB* build process is tested in the CI system to ensure compatiblity with the
+ CMS content schema.
+ 
+The *WEB* build process is triggered automatically by certain actions in the CMS 
+or manually via the [command line](#cli-build) or the [DevShop Dashboard](1).
+ 
+## Build Triggers
+The *WEB* instance of an environment is rebuilt when any of the following actions take place in the *CMS*:
+
+- Alert content is created or updated.
```
Comment (jefflbrauer) [inline READMES/environments.md]: Facility Alert or Individual Facility Operating Status is created or updated

File: READMES/environments.md [hunk 3]
```diff
@@ -1,19 +1,63 @@
...
+| [DEV](3) <br> Latest approved code.      | [dev.cms.va.gov](https://dev.cms.va.gov)             | [dev.cms.va.gov](https://dev.va.gov)                                                | [BRD: Jenkins](http://jenkins.vfs.va.gov/job/deploys/view/Dev/job/cms-vagov-dev/)
+| [LOCAL](4) <br> Local development        | [va-gov-cms.lndo.site](http://va-gov-cms.lndo.site)  | [va-gov-web.lndo.site](http://va-gov-web.lndo.site) <br> [va-gov-web.lndo.site/static](http://va-gov-web.lndo.site)  <br> [va-gov-web.lndo.site/$URL?_format=static_html](http://va-gov-web.lndo.site/$URL)  
+| [CI / PR](1)  <br> Pull Requests & Automated Testing           | pr###.ci.cms.va.gov     |  pr###.web.ci.cms.va.gov <br>  pr###.ci.cms.va.gov/static <br> pr###.ci.cms.va.gov/$URL?_format=static_html  | [CMS-CI: DevShop](1)     
+| [Demos](2) <br> Demos & Training       | NAME.demo.ci.cms.va.gov |  NAME.web.demo.ci.cms.va.gov  <br> NAME.demo.ci.cms.va.gov/static <br> NAME.demo.ci.cms.va.gov/$URL?_format=static_html | [CMS-CI: DevShop](1)
+
+
+## What is an Environment?
```
Comment (ElijahLynn) [inline READMES/environments.md]:
```suggestion
## What is an Environment? What is it really?
```

Comment (ElijahLynn) [inline READMES/environments.md]:
```suggestion
## What is an Environment? What is it really?
```

Comment (ElijahLynn) [inline READMES/environments.md]:
```suggestion
## What is an Environment? What is it really?
```

File: READMES/environments.md [hunk 4]
```diff
@@ -1,19 +1,129 @@
-# Environments
+# Environments & the Content Build Process
 
-Domains for this application are below, they both correspond to a frontend (FE) domain that consumes data from the CMS via GraphQL API endpoint at /graphql:
+In order to ensure quality and prevent problems from going to production, multiple
```
Comment (ElijahLynn) [inline READMES/environments.md]:
```suggestion
To enable end to end quality analysis, from the VA.gov CMS through to the public facing website, new feature review, and stakeholder demonstrations, multiple joined CMS/WEB environments may be created by any user with access to devshop.```

File: READMES/environments.md [hunk 5]
```diff
@@ -1,19 +1,129 @@
...
+ CMS content schema.
+ 
+The *WEB* build process is triggered automatically by certain actions in the CMS 
+or manually via the [command line](#cli-build) or the [DevShop Dashboard](1).
+ 
+### Build Triggers
+
+The *WEB* instance of an environment is rebuilt when any of the following actions take place in the *CMS*:
+
+- Alert content is created or updated.
```
Comment (ElijahLynn) [inline READMES/environments.md]:
```suggestion
- Facility Alert or Individual Facility Operating Status is created or updated
```

Comment (ElijahLynn) [inline READMES/environments.md]:
```suggestion
- Facility Alert or Individual Facility Operating Status is created or updated.
```

**Pull request merged**